hicommonwealth/commonwealth
# Decode Content in updateThread command

## Link to Issue
Closes: #9228 

## Description of Changes
- Decodes thread bodies in updateThread command
- Adds search vector updates in updateThread command
- Removed legacy update_thread code
- Removed legacy skipped thread tests

## Test Plan
- Create a thread
- Search for the thread
- Update the thread body
- Search for the new thread body
- Update the thread one last time to ensure the last edit did not mess up format.

## Deployment Plan

1. 

## Other Considerations

-
